src/setupTests.js

```diff
@@ -1,7 +1,5 @@
-/* eslint-disable linebreak-style */
 // jest-dom adds custom jest matchers for asserting on DOM nodes.
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-// eslint-disable-next-line linebreak-style
 import '@testing-library/jest-dom';
```

Comment:
- [ ] As required for the project, kindly make sure all the boilerplate that got created by the CRA (*texts*, *images*, *styles*,...) should be deleted i.e. all non related files, images,... to this project should deleted such as `src/setupTests.js`, `src/logo.svg`, `src/reports.js`,...